src/env-vars-validation-schema.ts

```diff
@@ -89,14 +89,15 @@ export const envVarsValidationSchema = Joi.object({
   ASSETS_SYNC_HISTORY_INTERVAL_IN_HOURS: Joi.number().positive().required(),
   ASSETS_SYNC_ENABLED: Joi.boolean().required(),
 
+  IPFS_CLIENT_URL: Joi.string().uri(),
+  IPFS_CLIENT_PROTO: Joi.string(),
```

Comment:
we will need to change `IPFS_PROTOCOL` in [gp4btc](https://github.com/energywebfoundation/gp4btc/blob/bdd5dd75b8172ef62e15025e2ea7096851c7ae7a/devops/gp4btc-ics-prod/values.yaml#L121) to `IPFS_CLIENT_PROTO`